config/locales/en.yml

```diff
@@ -560,8 +555,10 @@ en:
       programming_groups: Programming Groups
       finishing_rate: Finishing Rate
     submit:
-      failure: An error occurred while transmitting your score. Please try again later.
-      too_late: Your submission was saved successfully but was received after the deadline passed.
+      failure: The scoring was performed successfully but an error occurred while transmitting your score. Please try again later.
+      too_late: Your submission was saved successfully but was received after the deadline, so that only %{score_sent} points were transmitted.
```

Comment:
Should we add the redirect option `<a href="%{url}" class="alert-link">Please click here to finish the exercise now.</a>` here and also for `too_late_and_not_for_all_users_submitted`?

Reply:
Ideally, I would only add this link iff the user seeing the flash message was the one for which scoring worked. Currently, this case is not considered, as shown with the following screenshot (of course, that's me!):

<img width="1723" alt="Bildschirmfoto 2023-10-30 um 12 38 01" src="https://github.com/openHPI/codeocean/assets/7300329/f0998c53-256e-4536-8252-99064ae5ecab">

Reply:
Can one of these errors `IMS::LTI::XMLParseError, Net::OpenTimeout, Net::ReadTimeout, Errno::ECONNRESET, SocketError, EOFError` also occur if the lti_outcome_service is available?

Reply:
Those errors can _only_ occur if the `lti_outcome_service` is available. They can happen if the LMS is unavailable or returned an invalid XML `IMS::LTI::XMLParseError`...

Reply:
Question has been clarified, since now the reaching of the full score is independent of transmission errors. Also, we check if it is the current_user where errors occur.